element.io/nightly/config.json

```diff
@@ -47,5 +47,8 @@
     "posthog": {
         "projectApiKey": "phc_Jzsm6DTm6V2705zeU5dcNvQDlonOR68XvX2sh1sEOHO",
         "apiHost": "https://posthog.hss.element.io"
+    },
+    "features": {
+        "feature_spaces_metaspaces": true
```

Comment:
This will force it on with no UI path to disable, so we may want to be sure that is okay.

Reply:
After discussion it is okay for this specific lab, but we will need a way to have default-on labs via config.json without them being forced-on for IA1.3 and IA1.5